routes/volunteer_at_events/volunteer_at_events.js

```diff
@@ -0,0 +1,36 @@
+// endpoints related to volunteer_at_events
+const express = require('express');
+const pool = require('../../db');
+
+const volunteerAtEventsRouter = express();
+
+volunteerAtEventsRouter.use(express.json());
+
+// Get Volunteers By Event
+volunteerAtEventsRouter.get('/:id', async (req, res) => {
+  try {
+    const getVolunteerByEvent = await pool.query(
+      'SELECT user_id FROM volunteer_at_events WHERE event_id = $1;',
+      [req.params.id],
+    );
+    res.status(200).json(getVolunteerByEvent.rows);
+  } catch (err) {
+    res.status(400).json(err);
+  }
+});
+
+// Create Volunteer At Event
+volunteerAtEventsRouter.post('/create', async (req, res) => {
```

Comment:
move to volunteersRouter under route '/volunteers/event/join', don't forget to remove the '/volunteers' from the router.post() call